dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HadoopInitializerTask.java

```diff
@@ -35,10 +34,10 @@ public HadoopInitializerTask() {
   @Override
   protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
       throws Exception {
-    MetaHandle metaHandle = (MetaHandle) handle;
-    metaHandle.initializeConnector(
-        ClouderaMetadataConnector.CONNECTOR_NAME,
-        new ConnectorArguments("--connector", ClouderaMetadataConnector.CONNECTOR_NAME));
+
+    String connectorName = ClouderaMetadataConnector.CONNECTOR_NAME;
+    ConnectorArguments connectorArguments = new ConnectorArguments("--connector", connectorName);
```

Comment:
nit: can directly use `ClouderaMetadataConnector.CONNECTOR_NAME`

Reply:
The part of code just do nothing and will be rewritten in the next task. So, I think it's ok to leave it as is.